exdb/views.py

```diff
@@ -383,3 +385,36 @@ def get_context_data(self, *args, **kwargs):
         context = super(SearchExperienceResultsView, self).get_context_data(*args, **kwargs)
         context['search_query'] = self.request.GET.get('search', '')
         return context
+
+
+class SearchExperienceReport(CreateView):
```

Comment:
This can simply inherit from `View` since you don't use anything that `CreateView` has to offer.